guild/tests/flags-sub-commands.md

```diff
@@ -57,8 +57,8 @@ the `b` command, which is a Click group.
 
     >>> run("guild run click-b -y")
     base-foo=4
-    Usage: args_click.py b [OPTIONS] COMMAND [ARGS]...
-    Try 'args_click.py b --help' for help.
+    Usage: ...args_click... b [OPTIONS] COMMAND [ARGS]...
```

Comment:
I haven't gotten to the bottom of this one yet. I think I have isolated it to a change in click 7 vs 8, as opposed to any of the other code changes in this PR. I'll revert this change and fix this before merging this PR.